### PR TITLE
Aws task 3

### PR DESCRIPTION
**Pull request by DevandScorp**
What was done? 
Frontend application is integrated with product api (https://github.com/DevandScorp/aws-shop-frontend/pull/1)
FE deployment link: https://d3fkc8nqapw81s.cloudfront.net/
getProductsList https://fv8w9bbv2m.execute-api.eu-west-1.amazonaws.com/dev/products
getProductsById - https://fv8w9bbv2m.execute-api.eu-west-1.amazonaws.com/dev/products/{id}
Additional:
Async/await is used in lambda functions
+ ES6 modules are used for product-service implementation
+ Webpack is configured for product-service
- SWAGGER documentation is NOT created for product-service
+ Lambda handlers are covered by basic UNIT tests (NO infrastructure logic is needed to be covered)
+ Lambda handlers (getProductsList, getProductsById) code is written not in 1 single module (file) and separated in codebase.
+ Main error scenarious are handled by API ("Product not found" error, try catch blocks are used in lambda handlers).
Product schema :
Product {
    count: number,
    description: string,
    id: string,
    price: number,
    title: string,
    imageUrl: string
} 
TASK SCORE= 10/11
P.S: I changed my product-service to products-service because of serverless errors. When I changed my api manually in aws, serverless was unable to upload lambdas

### CODE REVIEW

File: products-service/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (kotmatpockuh) [inline products-service/.vscode/launch.json:1]: it's better to .gitignore this file

File: products-service/interfaces/Product.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export interface Product {
```
Comment (kotmatpockuh) [inline products-service/interfaces/Product.ts:1]: btw, I noticed your naming is different (I mean some files are capitalized, while others - no).

**Review by kotmatpockuh — APPROVED** [review]

**Review by svvald — APPROVED** [review]

File: products-service/functions/getProductById.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Products } from "../data/products";
+
+export const getProductById = (eventPath: string) => {
+    if (!eventPath) return { statusCode: 500, body: JSON.stringify({ message: 'Request path was not provided' }) }
+    const id = eventPath.slice(eventPath.lastIndexOf('/') + 1, eventPath.length);
```
Comment (artsiompeshko) [inline products-service/functions/getProductById.ts:5]: you should use event.pathParameters, no need for custom parsing...

**Review by artsiompeshko — APPROVED** [review]
10

**Pull request merged**